sdk/identity/Azure.Identity/src/ClientCertificateCredential.cs

```diff
@@ -10,18 +10,36 @@
 
 namespace Azure.Identity
 {
+    /// <summary>
+    /// Enables authentication of a service principal in to Azure Active Directory using a X509 certificate that is assigned to it's App Registration. More information
```

Comment:
>// [](start = 5, length = 2)

Should service principal be `Service Principal`?

Reply:
It's not capitalized in the AAD docs so I'm just following that convention.